recipes/uvw/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "uvw" %}
+{% set version = "2.12.1" %}
+{% set libuv_version = "1.44" %}
+{% set sha256 = "3460842778e91e7d2fae4201e49e4521e9d94fbbf3891ae6c52d3c9fc0673598" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/skypjack/uvw/archive/v{{ version }}_libuv_v{{ libuv_version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    - pkg-config
+  host:
+    - libuv
+  run:
+    - libuv
```

Comment:
```suggestion
    # Since libuv is lacking a run_exports, we need to pin the version used at runtime; see https://github.com/conda-forge/libuv-feedstock/pull/58
    - libuv
```